docs/libraries/azure-core/rules/no-generic-types.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "no-generic-types"
```

Comment:
```suggestion
title: "no-generic-numerics"
```

what do you think of this rename, `no-generic-types` makes me thinkg we don't allow templates specially for someone coming from `TS` or `C#`

Reply:
Updated.